### PR TITLE
Add neptune-release to the readme

### LINKED ISSUE #56: The README should be updated to mention the release tool
The new release tool is not mentioned at all in the README. Also at some point it might be nice to have a short tool page mentioning the main features.

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**
fixes #56
based on #62

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
Rebased so it does not depend on any other PR anymore (it shouldn't).

File: README.md [hunk 1]
```diff
@@ -32,8 +32,22 @@ The following documents are available:
...
+
+Fetches repositories for a selected GitHub organization and builds an HTML
 overview of applications and libraries that shows which version of each library
 is used in each application.
+
+
+[Release](https://github.com/sociomantic-tsunami/neptune/tree/master/src/release)
+-------
+
+Automates all the task required to do a semver abiding release, including some
```
Comment (gavin-norman-sociomantic) [inline README.md]: "all the tasks"

Comment (gavin-norman-sociomantic) [inline README.md]: "required to make a release abiding to the neptune specification,"

File: README.md [hunk 2]
```diff
@@ -32,8 +32,22 @@ The following documents are available:
...
+sanity checks. Task include:
```
Comment (gavin-norman-sociomantic) [inline README.md]: "Tasks"

File: README.md [hunk 3]
```diff
@@ -32,8 +32,22 @@ The following documents are available:
...
+
+* Autodetection of what type of release is being done (major, minor, patch)
+* Merging of features and bugfixes into higher versions
+* Creation of releases, including their tags, releasenotes and github releases
```
Comment (gavin-norman-sociomantic) [inline README.md]: "release notes"

**Comment by mathias-baumann-sociomantic** [comment]
> Rebased so it does not depend on any other PR anymore (it shouldn't).

Well, it does depend on them in so far as the features described are only implemented in those PRs

**Comment by mathias-baumann-sociomantic** [comment]
Pushed gavins suggestions

**Comment by mihails-strasuns-sociomantic** [comment]
> Well, it does depend on them in so far as the features described are only implemented in those PRs

Mentioning it in comment and marking as blocked is enough for that ;)

**Comment by mathias-baumann-sociomantic** [comment]
No longer blocked

**Comment by mathias-baumann-sociomantic** [comment]
rebased on current master even

**Pull request merged**